src/pudl/etl/__init__.py

```diff
@@ -31,6 +39,7 @@
     ferc_to_sqlite_settings,
 )
 from pudl.settings import EtlSettings
+from pudl.workspace.setup import PudlPaths
```

Comment:
This import also *seems* extraneous but may be sneakily needed somehow?

Reply:
No also a hold over from prototyping. Fixed now.